src/main/java/pw/kaboom/extras/modules/checks/DenyWaterFlow.java

```diff
@@ -0,0 +1,25 @@
+package pw.kaboom.extras.modules.checks;
+
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockFromToEvent;
+import org.bukkit.plugin.java.JavaPlugin;
+import pw.kaboom.extras.Main;
+
+public class DenyWaterFlow implements Listener {
+    private static final int max = JavaPlugin.getPlugin(Main.class).getConfig().getInt("maxWaterFlow");
+    private int waterFlowCount = 0;

```

Comment:
The liquid flow counter is never depleted, meaning that if the server hits the limit, water and lava will never be able to flow again, even if the server is well within the limits.